README.md

```diff
@@ -4,4 +4,63 @@
 
 # Matryoshka
 
-This library provides generalized folds, unfolds, and traversals for fixpoint data structures in Scala.
+This library provides generalized folds, unfolds, and traversals for fixed point data structures in Scala.
+
+## External Resources
+
+- [Functional Programming with Bananas, Lenses, Envelopes and Barbed Wire](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.41.125) – the iconic paper that collected a lot of this info for the first time
+- [Recursion Schemes: A Field Guide (Redux)](http://comonad.com/reader/2009/recursion-schemes/) – Ed Kmett’s summary (with links to Haskell code) of various folds and unfolds
+- [Unifying Structured Recursion Schemes](http://www.cs.ox.ac.uk/people/jeremy.gibbons/publications/urs.pdf) – a newer paper on how to generalize recursion schemes
+- [Efficient Nanopass Compilers using Cats and Matryoshka](https://github.com/sellout/recursion-scheme-talk/blob/master/nanopass-compiler-talk.org) – Greg Pfeil’s (Scala) talk on this library (and some other things)
+- [Fix Haskell (by eliminating recursion)](https://github.com/sellout/recursion-scheme-talk/blob/master/recursion-scheme-talk.org) – Greg Pfeil’s (Haskell) talk on recursion schemes
+
+## Introduction
+
+This library is predicated on the idea of rewriting your recursive data structures as functors where the recursive component is replaced by a parameter.
```

Comment:
Can this be said without using the word "functor"? It's not obvious to me what definition of the term is intended, and you don't yourself define it until the next section.

Also, "the recursive component is replaced by a parameter" is confusing because the _component_ is _always_ a parameter of the (value) constructor, whereas what you mean is something like "the type of the recursive component is given by a type parameter."